README.md

```diff
@@ -15,6 +27,7 @@ DP refers to Demographic Parity, EO to Equalized Odds, and BGL to Bounded Group
 
 To request additional algorithms or fairness definitions, please open a new issue.
 
+# <a name="existing"></a>
```

Comment:
Small thing.... can you do
`<a name="existing" />`

Reply:
why is that preferred over a properly terminated </a>?

Reply:
In theory, that should be properly terminated, as a self-closing tag. However, I don't have very strong opinions.